packages/eslint-plugin-pf-codemods/src/rules/v6/menuItemActionWarnUpdateMarkup/menuItemAction-warn-update-markup.ts

```diff
@@ -0,0 +1,35 @@
+import { getFromPackage } from "../../helpers";
+import { Rule } from "eslint";
+import { ImportDeclaration } from "estree-jsx";
+
+// https://github.com/patternfly/patternfly-react/pull/10089
+module.exports = {
+  meta: {},
+  create: function (context: Rule.RuleContext) {
+    const { imports } = getFromPackage(context, "@patternfly/react-core");
+
+    const menuItemActionImport = imports.find(
+      (specifier) => specifier.imported.name === "MenuItemAction"
+    );
+
+    return !menuItemActionImport
+      ? {}
+      : {
+          ImportDeclaration(node: ImportDeclaration) {
+            if (
+              node.specifiers.find(
+                (specifier) =>
+                  specifier.type === "ImportSpecifier" &&
```

Comment:
I was thinking about what if someone imports Patternfly and uses it this way:

```
import * as PF from '@patternfly/react-core';

<PF.MenuItemAction></PF.MenuItemAction>
```
I hope nobody does it, I checked the `getFromPackage` helper and we don't handle this `ImportNamespaceSpecifier` there either. So if we don't know of any team who would do it, we don't have to worry about it, it would bring too many complications.

Reply:
Ah that's an interesting point! I don't think we had any consumers bring this to our attention during the v5 release (@wise-king-sullyman do you recall if we did?), but worth keeping an eye out for with v6 alpha and beta releases

Reply:
That's in the same realm as the root problem behind #541, IMO it is probably worth us properly handling all of those import cases, but I don't think ironing those edge cases out is worth holding up the alpha release of the mods.

I think TS will be a big help with these kinds of problems, typing `getFromPackage` made it pretty obvious that we aren't logically setup for all of the potential import declarations.